temporalcli/commands.taskqueue_test.go

```diff
@@ -2,11 +2,12 @@ package temporalcli_test
 
 import (
 	"encoding/json"
-	"github.com/stretchr/testify/assert"
-	"go.temporal.io/sdk/workflow"
 	"strings"
 	"time"
 
+	"github.com/stretchr/testify/assert"
+	"go.temporal.io/sdk/workflow"
+
```

Comment:
Some of these changes are confusing to me showing up in the PR. So this change is already in `next-server`, why is it showing up in the PR?